juju/juju
# Implement Ping for vsphere

This change updates the stub Ping method to actually attempt
a connection and login to VSphere in order to be more
likely to discover when the user has typoed the endpoint.

QA: 
1. Build juju client
1. connect to canonical VPN
1. scp juju to oil-vsphere
1. run juju add-cloud and add a vsphere cloud with IP 10.245.0.134
1. it should work.  
1. Try it with a bad ip or ip of non-vsphere machine, should fail.
